pyquil/_parser/PyQuilListener.py

```diff
@@ -339,4 +339,4 @@ def _real(real):
     elif real.INT():
         return int(real.getText())
     else:
-        raise RuntimeError("Unexpected real: " + real.getText())
+        raise RuntimeError("Unexpected real: " + real.getText())  # pragma: no cover
```

Comment:
what do these pragmas do again?

Reply:
They mark lines that cannot be covered by the unit tests. At the top we have a conditional import, so only one block or the other will get run. And in the other cases, these are error conditions that can't currently happen as far as I can tell.